src/styles/main.scss

```
@@ -1,3 +1,58 @@
-body {
-  margin: 0;
+@import "./reset";
+@import "./tools/variables";
+@import "./tools/mixin";
+
+.main {
+  display: flex;
+  justify-content: center;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: $gap-size;
+  padding-top: 40px;
+
+  &--start-day {
+    @each $day, $day-count in $days-in-week {
+      &--#{$day} :first-child {
+        margin-left: ($day-count) * $day-margin;
+      }
+    }
+  }
+
+  &__day {
+    font-style: arial, helvetica;
+    font-size: 30px;
+    @include square($cell-size);
+    background-color: $color-white;
+    border: 1px solid $color-black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: 0.5s;
+
```

```suggestion
    display: flex;
    justify-content: center;
    align-items: center;
    @include square($cell-size);
    
    background-color: $color-white;
    border: 1px solid $color-black;
   
    transition: 0.5s;

```
Check order for styles